app/javascript/vue/protocol/container.vue

```diff
@@ -206,6 +206,9 @@
     created() {
       $.get(this.protocolUrl, (result) => {
         this.protocol = result.data;
+        this.$nextTick(() => {
+          this.refreshProtocolStatus();
+        });
```

Comment:
It would be better to move this to the mounted() hook (probably won't need $nextTick in that case.